code/modules/shuttle/on_move.dm

```diff
@@ -1,18 +1,34 @@
+//These defines are to make it easier to change the args available without having to copy paste it in everywhere
+//You need to also change the same procs in shuttle.dm
+//These are undefined at the end of the file
+#define TURF_FROM fromShuttleMove(turf/newT, turf_type, baseturf_type)
+#define TURF_TO toShuttleMove(turf/oldT, shuttle_dir)
+#define TURF_ON onShuttleMove(turf/newT, turf_type, baseturf_type, rotation, list/movement_force, move_dir)
+#define TURF_AFTER afterShuttleMove(turf/oldT)
+
+#define ATOM_BEFORE beforeShuttleMove(turf/newT, rotation)
+#define ATOM_ON onShuttleMove(turf/newT, turf/oldT, rotation, list/movement_force, move_dir, old_dock)
+#define ATOM_AFTER afterShuttleMove(list/movement_force, shuttle_dir, shuttle_preferred_direction, move_dir)
+
+#define AREA_BEFORE beforeShuttleMove()
+#define AREA_ON onShuttleMove(turf/oldT, turf/newT, area/underlying_old_area)
+#define AREA_AFTER afterShuttleMove()
```

Comment:
You can always just find-and-replace the defines yourself with the real shit anyway. You keep your convenience, we keep our readability, everyone wins